server/backend/backend.go

```diff
@@ -159,6 +179,8 @@ func (self *Backend) GetAddressByHash(ctx context.Context, hash string) (*models
 	}
 	address.BalanceWei = balance.String() //to make sure that we are showing most recent balance even if db is outdated
 	address.BalanceString = new(big.Rat).SetFrac(balance, wei).FloatString(18)
+	// todo: only store a subset of this data in the cache, just the metadata like token name, decimals, etc. Things that won't change.
```

Comment:
Is this a blocker for this PR? Or is there some kind of default timeout built in to the cache so that we'll at least be eventually consistent?

Reply:
Not a blocker since it's only used in that one function for now and it doesn't use the values that can change. If we start using these cache entries for the values that can change, then we'll want to update this.